reana_cluster/configurations/reana-cluster.yaml

```diff
@@ -34,6 +34,5 @@ components:
     image: "reanahub/reana-workflow-engine-yadage:0.1.0"
     environment:
       - ZMQ_PROXY_CONNECT: "tcp://zeromq-msg-proxy.default.svc.cluster.local:8666"
-      - QUEUE_ENV: "yadage-default-queue"
       - REANA_WORKFLOW_ENGINE_YADAGE_EXPERIMENT: "default"
```

Comment:
we should also remove this env variable as it is being set twice as well, see https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/backends/kubernetes/templates/deployments/workers-template.yaml#L21